### PR TITLE
Add support for enableTranscoding

### PR DESCRIPTION
**Pull request by biglittlebigben**

### CODE REVIEW

File: src/IngressClient.ts [hunk 1]
```diff
@@ -137,6 +148,7 @@ export class IngressClient extends ServiceBase {
       participantName = opts.participantName || '';
       participantIdentity = opts.participantIdentity || '';
       bypassTranscoding = opts.bypassTranscoding || false;
+      enableTranscoding = opts.enableTranscoding;
```
[the remark this answers is not in the document]
Comment (biglittlebigben) [reply src/IngressClient.ts:153]: Indeed, thanks

**Comment by biglittlebigben** [comment]
This is still pending on a protocol PR getting merged. How are protocol changes imported in the js SDK currently?

**Comment by lukasIO** [comment]
it's done by linking to the `@livekit/protocol` package that we publish from the protocol repo. I can cut a release for that now

**Comment by lukasIO** [comment]
ah wait, [your change](https://github.com/livekit/protocol/pull/680) isn't merged into main yet, so we'll have to wait for that and then update the `@livekit/protocol` dependency in this repo.

**Review by lukasIO — APPROVED** [review]
lgtm!

**Pull request merged**